syft/frameworks/torch/hook.py

```diff
@@ -805,7 +863,7 @@ def _add_methods_from__torch_tensor(tensor_type: type, syft_type: type):
             "__sizeof__",
             "__subclasshook__",
             "_get_type",
-            "__eq__",
+            # "__eq__", # FIXME it now overwritten in native.py to use torch.eq, because of pb between == & __eq__
```

Comment:
Can you create an issue for this with more details

Reply:
#2030

Reply:
Can you also link the issue in the comment